sematic/ui/src/components/RunNode.tsx

```diff
@@ -11,7 +11,18 @@ import { Handle, NodeProps, Position } from "react-flow-renderer";
 import { Run } from "../Models";
 import RunStateChip from "./RunStateChip";
 
-function getColor(futureState: string, theme: Theme): PaletteColor {
+function getColor(
+    futureState: string, isCloned: boolean, theme: Theme
+): PaletteColor {
+  var greyPaletteColor = {
+    light: theme.palette.grey[400],
+    dark: theme.palette.grey[600],
+    main: theme.palette.grey[400],
+    contrastText: "",
+  }
+  if (isCloned) {
+    return greyPaletteColor;
+  }
```

Comment:
Nit: It looks like we might need several things from the run; setting the first parameter of type `Run` might make things easier and more extensible. 

suggestion:
```typescript
function getColor(
    {future_state, original_run_id}: Run, theme: Theme
): PaletteColor {
  var greyPaletteColor = {
    light: theme.palette.grey[400],
    dark: theme.palette.grey[600],
    main: theme.palette.grey[400],
    contrastText: "",
  }
  if (!!original_run_id) {
    return greyPaletteColor;
  }
  if (future_state === "RESOLVED") {
    return theme.palette.success;
  }
  if (future_state === "RETRYING") {
    return theme.palette.warning;
  }
  if (["SCHEDULED", "RAN"].includes(future_state)) {
    return theme.palette.info;
  }
  if (["FAILED", "NESTED_FAILED"].includes(future_state)) {
    return theme.palette.error;
  }
  return greyPaletteColor;
}
```

This is a nit. You don't have to change if you feel that doesn't make sense

Reply:
Actually, I would rather we do not change the treatment in the DAG view at this time.

I would prefer keeping the color treatment as is for now (it is already a little too colorful, I want to avoid more variation). To denote a cloned run, I would rather display an icon such as e.g. https://mui.com/material-ui/material-icons/?selected=JoinFull
either to replace the run status dot, or in the top right corner of the node. The icon can have a hover tooltip saying "Run output cached from run <ID>".

Not required for this PR, can be a follow up. For this PR, you can just leave the original treatment with no distinction for cached/cloned runs.